src/transforms/transform_dateFormat.stories.mdx

```diff
@@ -0,0 +1,27 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import { transform_dateFormat } from "./column_transforms";
+import { TransformExample } from "./utils/TransformExample";
+
+<Meta title="Utils/Transforms/DateFormat"/>
+
+# transform_dateFormat
+
+Convert a Date object into a month / day / year string.
```

Comment:
Just a spelling suggestion, conjugate the convert verb in the third person like the following:

**Converts** a Date object into a month / day / year string.